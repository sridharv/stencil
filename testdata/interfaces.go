package ifaces

type holder struct {
	data interface{}
}

type Set map[interface{}]struct{}

func (s Set) Add(a interface{}) {
	s[a] = struct{}{}
}

func (s Set) Delete(a interface{}) {
	delete(s, a)
}
