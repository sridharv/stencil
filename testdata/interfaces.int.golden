package ifaces

type holder struct {
	data int
}

type Set map[int]struct{}

func (s Set) Add(a int) {
	s[a] = struct{}{}
}

func (s Set) Delete(a int) {
	delete(s, a)
}
